src/d3a/d3a_core/sim_results/endpoint_buffer.py

```diff
@@ -87,7 +87,7 @@ def generate_json_report(self):
             "unmatched_loads": self.unmatched_loads,
             "cumulative_loads": self.cumulative_loads,
             "price_energy_day": self.price_energy_day,
-            "cumulative_grid_trades": self.cumulative_grid_trades,
+            "cumulative_grid_trades": self.cumulative_grid_trades_redis,
```

Comment:
Is it a remnant, or have we decided to use the redis output for the CSV files too ?

Reply:
Wanted to test redis output. Will remove it now. Thanks for pointing it out.